src/languages/lsl.js

```diff
@@ -0,0 +1,133 @@
+/*
+    Language: Linden Scripting Language
+    Description: The Linden Scripting Language is used in Second Life by Linden Labs.
+    Author: Builder's Brewery <buildersbrewery@gmail.com>
+    Category: common, scripting
+*/
+
+function(hljs) {
+    return {
+        case_insensitive: false,
+        aliases: ['lsl'],
+        illegal: ':',
+        contains: [
+            {
+                className: 'string',
+                begin: '"',
+                end: '"'
+            },
+            {
+                className: 'comment',
+                begin: '//',
+                end: '$'
+            },
+            {
+                className: 'comment',
+                begin: '/\\*',
+                end: '\\*/'
+            },
+            {
+                className: 'number',
+                begin: '(\\b0[xX][a-fA-F0-9]+|(\\b\\d+(\\.\\d*)?|\\.\\d+)([eE][-+]?\\d+)?)'
+            },
+            {
+                className: 'state',
```

Comment:
We use a pre-defined set of class names to ensure that all languages work well with all the styles. The philosophy is explained in the [style guide](http://highlightjs.readthedocs.io/en/latest/style-guide.html) but it boils down to simply using only class names from http://highlightjs.readthedocs.io/en/latest/css-classes-reference.html

On a related note, the CSS file you provided in this pull request is not needed per se, unless you plan to turn it into a general (as opposed to language specific) theme.